CSUMB-SP17-CST499/group-assignment-tool
# Updated HTML to Jinja syntax and corrected dropdown menu problem

**Description**:  I translated our HTML files in the template folder into jinja syntax and removed the CSS that was causing the problems with the drop-down menu. I also removed the bootstrap files we had in the repo since we are using the bootstrap from CDN. 

**Testing**:  Steps to verify
1. Verify all templates that have endpoints in the app.py appear on the screen when visited.
2. Verify the drop-down menu in each of the templates from the previous step work (The pages aren't linked yet, so just verify the dropdown has the proper behavior)
